### PR TITLE
[JW Player Video Provider][Video JS Video Provider]: Add missing typedefs

### PR DESCRIPTION
**Pull request by karimMourra**
## Type of change
- [x] Code style update (formatting, local variables)

## Description of change
Adds missing typedefs for types that are referenced to avoid linter warnings.
It would be nice if the types could be shared to reduce duplication

### CODE REVIEW

**Comment by patmmccann** [comment]
It is possible to import types, @chernodub did it here

https://github.com/prebid/Prebid.js/blob/master/src/types/ortb2.d.ts

**Comment by patmmccann** [comment]
https://github.com/prebid/Prebid.js/pull/11902/files is also importing types

File: modules/videojsVideoProvider.js [hunk 1]
```diff
@@ -13,9 +13,6 @@ import { submodule } from '../src/hook.js';
 import stateFactory from '../libraries/video/shared/state.js';
 import { PLAYBACK_MODE } from '../libraries/video/constants/constants.js';
 import { getEventHandler } from '../libraries/video/shared/eventHandler.js';
-/**
- * @typedef {import('../libraries/video/shared/state.js').State} State
```
Comment (patmmccann) [inline modules/videojsVideoProvider.js]: Why revert this? looks like it worked

Comment (karimMourra) [reply modules/videojsVideoProvider.js]: THe duplication check is buggy . I tried to see if reducing the amount of files changed could limit its scope, but that's not the case.

**Comment by patmmccann** [comment]
Thanks!

**Review by patmmccann — APPROVED** [review]

**Pull request merged**